device/mellanox/x86_64-mlnx_msn4800-r0/pcie.yaml

```diff
@@ -0,0 +1 @@
+../x86_64-mlnx_msn4600c-r0/pcie.yaml
```

Comment:
Did you get it confirmed that this file can be re-used from 4600c?

Reply:
file was taken according to the anaconda coffee lake system.
more changes may be needed when the new system will be arrived